File: linear_search.py
def linear_seacrh(array, target):
    for i in range(len(array)):
        if array[i] == target:
            return i
    return "Element not found"

def verify(index):
    if index == "Element not found":
        print("Element not found in the array.")
    else:
        print(f"Element found at index: {index}")

array = [1, 2, 3, 4, 5]
target = 3
result = linear_seacrh(array, target)
verify(result)

File: linear_search.js
let array = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10];

function linearSearch(array, target){
    for(i = 0; i<array.length; i++){
        if(array[i] === target){
            return i
            ; // Return the index of the target element
        }
    }
    return -1; // Return -1 if the target element is not found
}

function verify(index){
    if(index === -1){
        return console.log("Element not found");
    }
    else{
        return console.log(`Element found at index: ${index}`); // Log the index of the found element
    }
}

result =linearSearch(array, 4);
verify(result); // Example usage, searching for the number 11